Added constraints for platform email sending. (#3020)
SoftwareAG/c8y-docs: content/service-terms/service-level-bundle/platform-sla.md

    * **Third-party connectivity services:** Connectivity may involve third-party services such as LPWAN or mobile network operators. Customer acknowledges that while {{< product-c8y-iot >}} facilitates the transfer of data through these services, it does not operate, monitor, or troubleshoot these third-party networks. Connectivity between Customer's devices and {{< product-c8y-iot >}} service is in the sole responsibility of Customer.
* **Data recovery**: While {{< company-c8y >}} maintains backups of data for its own business continuity management, disaster recovery on behalf of Customer (for example, after accidental data deletion by Customer) is a separate service. Customer expressly acknowledges the backup retention period and RPO outline above.

> * **eMail notification**:  {{< company-c8y >}} provides basic notification service via standard SMTP (unencrypted, port 25) for their customers to use via {{< company-c8y >}} infrastructure components (SMTP relay). This infrastructure has been setup to transfer emails using {{< company-c8y >}} domain, including e.g. correct SPF records for this domain. Customers using an enterprise tenant can configure their own eMail domain and relay and use e.g. encrypted SMTP (STARTTLS) or SMTPS with their own SMTP relay. If customer uses their own email domain, it is responsibiltiy of the customer to configure e.g. correct SPF records for that domain.

Reviewer: ```suggestion
* **Email notification**:  {{< company-c8y >}} provides a basic notification service via standard SMTP (unencrypted, port 25) for their customers to use via the {{< company-c8y >}} infrastructure components (SMTP relay). This infrastructure has been setup to transfer emails using the {{< company-c8y >}} domain, including, for example, correct SPF records for this domain. Customers using an {{< enterprise-tenant>}} can configure their own email domain and relay and use, for example, encrypted SMTP (STARTTLS) or SMTPS with their own SMTP relay. If customer uses their own email domain, it is in the responsibility of the customer to configure, for example, the correct SPF records for that domain.
```

> * **Email notification**:  {{< company-c8y >}} provides a basic notification service via standard SMTP (unencrypted, port 25) for their customers to use via the {{< company-c8y >}} infrastructure components (SMTP relay). This infrastructure has been setup to transfer emails using the {{< company-c8y >}} domain, including, for example, correct SPF records for this domain. Customers using an {{< enterprise-tenant>}} can configure their own email domain and relay and use, for example, encrypted SMTP (STARTTLS) or SMTPS with their own SMTP relay. If customer uses their own email domain, it is in the responsibility of the customer to configure, for example, the correct SPF records for that domain.

Reviewer 2: Who is "their"?

Reviewer: @eickler I assume, Cumulocity :-)? Shall I merge this as is or do we need a change here?

Reviewer 2: @BeateRixen I guess so ... that's why I used the capital "Customer" in most cases. I changed it and let's see.

Author: Fine with me in any case.
